string req
---
int64 obj_x
int64 obj_y
